src/store/messages-slice.ts

```diff
@@ -154,6 +167,7 @@ export const messagesSlice = createSlice({
 		builder.addCase(getConv.fulfilled, produce(getConvFulfilled));
 		builder.addCase(saveDraft.fulfilled, produce(saveDraftFulfilled));
 		builder.addCase(search.fulfilled, produce(fetchConversationsFulfilled));
+		builder.addCase(deleteAllAttachments.fulfilled, produce(deleteAttachmentsFulfilled));
```

Comment:
It is correct but change the name to deleteAttachments instead of deleteAllAttachments